addons/mail/static/src/web/activity/activity.js

```diff
@@ -72,6 +72,10 @@ export class Activity extends Component {
         return computeDelay(this.props.data.date_deadline);
     }
 
+    get onUpdate() {
+        return this.props.onUpdate();
```

Comment:
`onUpdate` props is optional, so would `this.props.onUpdate?.()` just to be sure

Reply:
Oh right, done!